pkg/services/sqlstore/team.go

```diff
@@ -292,9 +285,10 @@ func (ss *SQLStore) GetTeamById(ctx context.Context, query *models.GetTeamByIdQu
 	for _, user := range filteredUsers {
 		params = append(params, user)
 	}
+	params = append(params, query.UserIdFilter)
 
 	if query.UserIdFilter != models.FilterIgnoreUser {
-		sql.WriteString(` INNER JOIN team_member ON team.id = team_member.team_id AND team_member.user_id = ?`)
+		sql.WriteString(` AND team_member.user_id = ?`)
```

Comment:
```suggestion
		sql.WriteString(` WHERE team_member.user_id = ?`)
```

Reply:
this needs to be WHERE because we don't have anything else starting the where clause here.  AND will "work" but will get tacked onto the end of the join ON condition and won't have any effect with the switch to LEFT join